docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -6,7 +6,13 @@ This page describes the process followed for "core" releases (mainly the `framew
 
 ## Release Planning
 
-Our most up-to-date release plans are typically in the ["framework" milestone](https://github.com/silverstripe/silverstripe-framework/milestones) and ["cms" milestone](https://github.com/silverstripe/silverstripe-cms/milestones).
+Releases are planned by assigning issues to Github milestones.
+SilverStripe is split up into many modules listed on [github.com/silverstripe](github.com/silverstripe).
```

Comment:
I'm not sure whether protocol-less URLs will be parsed correctly. Might be safer to add it just in case?